updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc

```diff
@@ -0,0 +1,136 @@
+include::_attributes/common-attributes.adoc[]
+:context: updating-restricted-network-cluster-OSUS
+
+toc::[]
+
+[id="update-restricted-network-cluster-update-service"]
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
+
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information.
+
+To provide a similar update experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
+
+The following sections describe how to provide over-the-air updates for your disconnected cluster and its underlying operating system.
+
+[id="update-service-prereqs"]
+== Prerequisites
+
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry must be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
+* For more information on installing Operators, see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+== Configuring access to a secured registry for the OpenShift update service
+
+If the release images are contained in a secure registry, complete the steps in xref:../../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
+
+The OpenShift Update Service Operator needs the config map key name `updateservice-registry` in the registry CA cert.
+
+.Image registry CA config map example for the update service
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-registry-ca
+data:
+  updateservice-registry: | <1>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+  registry-with-port.example.com..5000: | <2>
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+----
+<1> The OpenShift Update Service Operator requires the config map key name updateservice-registry in the registry CA cert.
+<2>  If the registry has the port, such as `registry-with-port.example.com:5000`, `:` should be replaced with `..`.
+
+include::modules/images-update-global-pull-secret.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the OpenShift Update Service Operator
```

Comment:
It is not clear to me why we need an inline section on this, and can't just include "Have the OpenShift Update Service Operator installed" as a pre-requisite.  But if we're looking to limit the immediate change in this pull request, and possibly consider that sort of consolidation in follow-up work, that's fine with me.

Reply:
Good idea, though similar to @shellyyang1989 's suggestion to flesh out a new prereq page and simplify the actual updating content, I think I'll defer this change to that effort